fund_store/config/fund_loader_config/FAB/__init__.py

```diff
@@ -50,14 +51,34 @@
             print("No round config found in the loader config.")
             raise ValueError(f"No round_config found in {file}")
         fund_short_name = loader_config["fund_config"]["short_name"]
-        round_short_name = loader_config["round_config"]["short_name"]
 
         # Ensure the fund exists in the main config
         if fund_short_name not in FAB_FUND_ROUND_CONFIGS:
             FAB_FUND_ROUND_CONFIGS[fund_short_name] = loader_config["fund_config"]
             FAB_FUND_ROUND_CONFIGS[fund_short_name]["rounds"] = {}
-        FAB_FUND_ROUND_CONFIGS[fund_short_name]["rounds"][round_short_name] = loader_config["round_config"]
-        FAB_FUND_ROUND_CONFIGS[fund_short_name]["rounds"][round_short_name]["sections_config"] = loader_config[
-            "sections_config"
-        ]
-        FAB_FUND_ROUND_CONFIGS[fund_short_name]["rounds"][round_short_name]["base_path"] = loader_config["base_path"]
+        if isinstance(loader_config["round_config"], dict):
+            round_short_name = loader_config["round_config"]["short_name"]
+            FAB_FUND_ROUND_CONFIGS[fund_short_name]["rounds"][round_short_name] = loader_config["round_config"]
+            FAB_FUND_ROUND_CONFIGS[fund_short_name]["rounds"][round_short_name]["sections_config"] = loader_config[
+                "sections_config"
+            ]
+            FAB_FUND_ROUND_CONFIGS[fund_short_name]["rounds"][round_short_name]["base_path"] = loader_config[
+                "base_path"
+            ]
+
+        # Allows for one file per fund and not redefining the fund information each time
+        if isinstance(loader_config["round_config"], list):
```

Comment:
Looks like a good workaround to me, if these were here for the long term I'd suggest just having the list syntax by default and updating the others to have one dictionary in a list (the assessment configs do some light touch schema validation I think which gives some assurance when reviewing config changes!)

Hopefully all of this goes into environment config in databases and can move out of repositories soon enough

Reply:
Yeah, I didn't refactor fully as hoping this will be gone soon 🤞